js/global.d.ts

```diff
@@ -98,6 +112,14 @@ declare type RuntimeInfo = {
     igor_path: string;
 };
 
+/**
+ * Information for a specific found user.
+ */
+declare type UserInfo = {
```

Comment:
The next exciting thing from this is it opens up reading `devices.json`, so happy to see this!